ipatests/test_integration/test_cert.py

```diff
@@ -46,3 +47,50 @@ def test_cacert_file_appear_with_option_F(self):
         assert status == "MONITORING"
 
         self.clients[0].run_command(['ls', '-l', '/etc/pki/tls/test.CA'])
+
+
+class TestInstallMasterReplica(IntegrationTest):
+    num_replicas = 1
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master)
+        tasks.install_replica(self.master, self.replicas[2], setup_ca=True,
```

Comment:
Only one replica is defined for the class. It should be `cls.replicas[0]`. Also you need to change `self.master` to `cls.master`

Reply:
Just fixed it. Realized it after checking it in. Thanks